JohnDoddy/iReporter-Project
# #162223694 Finishes get all redflags

**What does this PR do?**
`This PR creates the endpoints to get all red flags`

**Description of Task to be completed?**
`Ensure the app is getting all red flag`
**How should this be manually tested?**
```
create a virtual environment 
activate the virtual environment
install requirements using pip install -r requirements.txt
run pytest n the test for getting all should pass
or run the app via python run.py and test on postman
```
